<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleItem</class>
 <widget class="QWidget" name="SaleItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget#SaleItem { background-color: transparent; }
    QLabel { font-family: &quot;Segoe UI&quot;; }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="container">
     <property name="styleSheet">
      <string notr="true">
        QFrame#container {
            background-color: white;
            border-bottom: 1px solid #F1F5F9; /* Slate-50 Divider */
            border-radius: 0px; /* Flat list style like React Table */
        }
        QFrame#container:hover {
            background-color: #F8FAFC; /* Hover effect (Slate-50) */
        }
      </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_time">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>color: #475569; font-family: &quot;Consolas&quot;; font-size: 13px; border: none;</string>
        </property>
        <property name="text">
         <string>10:45 AM</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerLeft">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_items">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
           background-color: #F1F5F9;
           color: #475569;
           border-radius: 10px;
           font-weight: bold;
           font-size: 12px;
           border: none;
         </string>
        </property>
        <property name="text">
         <string>2 items</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="fixedSize" stdset="0">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerRight">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_total">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>color: #1E293B; font-weight: bold; font-size: 14px; border: none;</string>
        </property>
        <property name="text">
         <string>â‚±85.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
